brand.go

```diff
@@ -0,0 +1,13 @@
+package easypost
+
+type Brand struct {
```

Comment:
Please add a comment above this line explaining what a brand is like all other structs have